src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
```

Comment:
Discussing at #IRCv3 it depends on IRCd configs etc. but we cannot be fully sure.

@DarthGandalf Could we put git users to test the modern compatibility and see if we have any issues and if not, continue with the modern compatibility?